common/esxi_manage_vmkernel_adapter.yml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Add or remove VMkernel network adapter on ESXi host to a vSphere Standard Switch (VSS)
+# port group or a vSphere Distributed Switch (VDS) port group
+# Paramters:
+#   vmk_device: The VMkernel network adapter name.
+#   vmk_state: The VMkernel network adapter state: present or absent.
+#   vmk_vss_name: The vSphere Standard Switch name for the VMkernel network adapter.
+#   vmk_vds_name: The vSphere Distributed Switch name for the VMkernel network adapter.
+#   vmk_portgroup_name: The port group name for the VMkernel network adapter.
+#   vmk_network: The network config for the VMkernel network adapter.
+#   vmk_enable_mgmt: True to enable management on the VMkernel network adapter.
+#     Default value if false.
```

Comment:
"Default value if false." ---> "Default value is false."

Reply:
Fixed